### PR TITLE
support sequence type; support opset 14 of identity op

### PR DESCRIPTION
**Pull request by chudegao**
Signed-off-by: degaochu <degaochu@cn.ibm.com>

### CODE REVIEW

File: onnx_tf/handlers/backend/identity.py [hunk 1]
```diff
@@ -20,3 +16,11 @@ def version_1(cls, node, **kwargs):
   @classmethod
   def version_13(cls, node, **kwargs):
     return [cls.make_tensor_from_onnx_node(node, **kwargs)]
+
+  @classmethod
+  def version_14(cls, node, **kwargs):
+    x = kwargs["tensor_dict"][node.inputs[0]]
+    if isinstance(x, (list, tuple)):
+      return [tf.identity_n(x)]
```
Comment (chinhuang007) [inline onnx_tf/handlers/backend/identity.py:24]: Please note ONNX sequence allows variable shapes and tf.identity_n supports tensors with same shapes only.

**Comment by chinhuang007** [comment]
In general, an ONNX sequence can have tensors with difference shapes. We should cover such case with code and tests to claim the complete support.
Currently the standard ONNX backend tests for identity with sequences are commented out in test/backend/test_onnx_backend.py, line 148. Once uncommented, we don't pass the standard tests with this patch.

**Comment by chudegao** [comment]
onnx backend test have an issue to support seqence output, I submit a PR to onnx to fix it https://github.com/onnx/onnx/pull/3563. After it's merged I will push new commit in this pr, otherwise test_identity_sequence will still fail.

**Review by chinhuang007 — APPROVED** [review]
LGTM!

**Pull request merged**